content/docs/api-reference/open.md

```diff
@@ -177,11 +174,7 @@ providing a `remote` argument:
 ```py
 import dvc.api
 
-with dvc.api.open(
-        'activity.log',
-        repo='location/of/dvc/project',
-        remote='my-s3-bucket'
-        ) as f:
+with dvc.api.open('activity.log', remote='my-s3-bucket') as f:
```

Comment:
Removed `repo` here for readability. It's about the `remote` argument, so I think it's okay to skip` repo=` kwarg.